xmppserver/src/main/java/org/jivesoftware/openfire/audit/spi/AuditorImpl.java

```diff
@@ -445,8 +440,8 @@ public Element getElement() {
          *
          * @return the date when the packet was audited.
          */
-        public Date getCreationDate() {
```

Comment:
Concerns of API break here?

Reply:
Although the _method_ is public, the _class_ is private. I think that makes it safe to change this API, as all consumers are limited to those that have visibility to this class (which is only the 'outer' class in the same file).